packages/lesswrong/components/review/BestOfLessWrongAdmin.tsx

```diff
@@ -0,0 +1,182 @@
+    ...theme.typography.body2,
+    '& h2': {
+      marginTop: 10,
+      marginBottom: 0,
+      position: 'sticky',
+      top: 0,
+      zIndex: theme.zIndexes.bestOfLessWrongAdminHeader,
+      backgroundColor: theme.palette.background.default,
+    }
+  },
+  post: {
+    ...theme.typography.body2,
+    ...theme.typography.commentStyle,
+    display: 'flex', 
+    gap: '10px',
+    alignItems: 'flex-start',
+    marginBottom: 30,
+    '& h3': {
+      marginBottom: 6,
+    }
+  },
+  postWrapper: {
+    display: "flex",
+    gap: '10px',
+    flexWrap: "wrap",
+    textWrap: "wrap",
+  },
+  generateButton: {
+    padding: '10px 0',
+    display: 'block',
+    color: theme.palette.primary.dark,
+    backgroundColor: 'transparent',
+    border: 'none',
+    fontWeight: 'bold',
+    cursor: 'pointer',
+    '&:disabled': {
+      opacity: 0.5,
+      cursor: 'not-allowed',
+    }
+  },
+  loadingText: {
+    marginLeft: 10,
+    color: theme.palette.grey[600],
+    fontStyle: 'italic',
+  }
+}));
+
+/* 
+Display the art for all ReviewWinners created in a given year,
+making it easier to quickly make a call for each post.
+
+(Note, because it was a bit annoying to do the fetch otherwise, this is using *createdAt*
+date rather than ReviewYear, i.e. 2 years after a ReviewYear)
+
+*/
+export const BestOfLessWrongAdmin = () => { 
+  const classes = useStyles(styles);
+  const { Loading } = Components;
+  
+  const currentUser = useCurrentUser();
+
+  const { data, loading: reviewWinnersLoading } = useQuery(gql`
+    query GetAllReviewWinners {
+      GetAllReviewWinners {
+        ...PostsTopItemInfo
+      }
+    }
+    ${fragmentTextForQuery('PostsTopItemInfo')}
+  `);
+  const reviewWinners: PostsTopItemInfo[] = data?.GetAllReviewWinners ?? [];
+  const reviewWinnersWithoutArt = reviewWinners.filter((reviewWinner: PostsTopItemInfo) => !reviewWinner.reviewWinner?.reviewWinnerArt);
+
+  const { params: { year } } = useLocation()
+
+  const { results: images, loading: imagesLoading, refetch: refetchImages } = useMulti({
+    collectionName: 'ReviewWinnerArts',
+    fragmentName: 'ReviewWinnerArtImages',
+    terms: {
+      view: 'allForYear',
+      year: parseInt(year),
+      limit: 5000,
+    },
+    skip: !year,
+  });
+  const groupedImages = groupBy(images, (image) => image.postId);
+  
+  if (!userIsAdmin(currentUser)) {
+    return <div>You are not authorized to view this page</div>
+  }
+
+  return <div className={classes.root}>
+    <h1>Best of LessWrong Admin</h1>
+    <p>Showing art created in {year}</p>
+      {(reviewWinnersLoading || imagesLoading) && <Loading/>}
+      <div>
+        <div>{Object.entries(groupedImages).length} Posts with art</div>
+        <div>{reviewWinnersWithoutArt.length} Posts without art</div>
+      </div>
+      <div>
+        {Object.entries(groupedImages).map(([title, images]) => {
+          const post = reviewWinners.find((reviewWinner: PostsTopItemInfo) => reviewWinner._id === title);
```

Comment:
`_id === title` are you sure this is right?

Reply:
(I see `const groupedImages = groupBy(images, (image) => image.postId);`, so I guess it is, but the it probably shouldn't be `title`)

Reply:
ah, I changed that in one place but not another